### DISCUSSION TITLE
Passwordless options

### DISCUSSION THREAD

**Discussion by paulincai** [meteor/meteor#12075] [Feature Requests]
1. Update the Meteor docs to include the options for Passwordless configuration.

2. Add 'loginTokenExpirationInMinutes'. Passwordless implies a "transactional" action: I want to login now, I need my token now. If I get access to someone's email (family members, colleagues at the office for instance) and get a valid token, I can access the app. I think by construction this should be limited to a maximum of 10 minutes (for security reasons indeed). I see no situation when I would need it for more than 5 minutes.

3. add "sequenceGenerator" in options where I can provide my own token generator function. To be frank, all the apps that implement passwordless I use, have 4-6 digits (numbers). On mobile, you can trigger the number keyboard to make it easy for the user instead of getting confused about whether capitals are required or not and chances for a misspelling are high when you have to memorize the code (between switching apps on the screen).

4. suggest in documentation for the user, to build a mechanism to prevent multiple attempts on using a code/token; for instance, maximum 3 attempts then a token is being invalidated. For this a new function is required for Passwordles which is "invalidateAllTokens" ... or something similar.

```javascript

Accounts.config({
   ...
   //  sendVerificationEmail: true,
   // passwordResetTokenExpirationInDays: 1,
   // defaultFieldSelector: {
   //   emails: 1
   // },
    tokenSequenceLength: 4,
    loginTokenExpirationMinutes: 5,
    sequenceGenerator: () => Math.floor(1000 + Math.random() * 9000)
  })

```

**Comment by denihs** [comment]
Hi @paulincai, everything here sounds good. 

We'll work on this.

Thank you!
